### PR TITLE
Flags locales

### PR DESCRIPTION
**Pull request by wyattjoh**
## What does this PR do?

- Flags now are all required to pre-define their types in the graph, or at the very least provide their translations in locales.

Ref: https://www.pivotaltracker.com/story/show/151179316

## How do I test this PR?

1. Create some flags
2. Check out the translated responses in the admin

### CODE REVIEW

File: graph/typeDefs.graphql [hunk 1]
```diff
@@ -948,7 +972,7 @@ input CreateFlagInput {
   item_type: ACTION_ITEM_TYPE!
 
   # The reason for flagging the item.
-  reason: String!
+  reason: FLAG_REASON
```
Comment (alinhle) [inline graph/typeDefs.graphql:975]: Required?

**Comment by alinhle** [comment]
We also need to adapt toxic comment for its TOXIC_COMMENT flag.

File: graph/typeDefs.graphql (hunk 1 above)
Comment (wyattjoh) [reply graph/typeDefs.graphql:975]: Nope!

**Review by alinhle — CHANGES_REQUESTED** [review]
image
The graphql refuses to load any system flags whose reasons are not defined in the `FLAG_REASON` type, e.g. trust, suspect and toxic comment flags

**Review by alinhle — DISMISSED** [review]

**Comment by wyattjoh** [comment]
Future PR will merge this branch in.

**Pull request closed without merging**